CMSgov/easi-app
# [EASI-2998] Pre-sign-in homepage

# EASI-2998

[Figma](https://www.figma.com/file/P1XsRsvCWr0OBWLa8lEcNP/EASi-Branding-and-Marketing?type=design&node-id=85-3367&t=zUfbvrys1epMmB70-0)

## Changes and Description

Homepage updates



## How to test this change

1. Sign out of EASi
2. Ensure homepage matches Figma
3. Check responsiveness



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
